### ISSUE TITLE
False Positive for W0602: global-variable-not-assigned

### ISSUE THREAD

**Issue by Chris-Kimmel** [PyCQA/pylint#4809]
### Bug description

```python
# pylint: disable=missing-module-docstring,missing-function-docstring
# pylint: disable=invalid-name,import-outside-toplevel
def import_it():
    global argp, namedtuple
    import argparse as argp
    from collections import namedtuple
```

### Command used

```shell
pylint a.py
```

### Pylint output

```shell
************* Module deleteme
deleteme.py:4:4: W0602: Using global for 'namedtuple' but no assignment is done (global-variable-not-assigned)
```

### Expected behavior

Pylint should NOT warn W0602 (global-variable-not-assigned)

### Pylint version

```shell
pylint 2.9.6
astroid 2.6.6
Python 3.9.6 | packaged by conda-forge | (default, Jul 11 2021, 03:39:48) 
[GCC 9.3.0]
```
